tests/api/test_account_bot.py

```diff
@@ -0,0 +1,335 @@
+import graphene
```

Comment:
Let's rename this file to `test_service_account.py` to match the naming used in the models/API.